crates/polars-ops/src/frame/pivot/mod.rs

```diff
@@ -157,12 +162,24 @@ where
         .into_iter()
         .map(|s| s.as_ref().to_string())
         .collect::<Vec<_>>();
-
+    let values = match values {
+        Some(v) => v
+            .into_iter()
+            .map(|s| s.as_ref().to_string())
+            .collect::<Vec<_>>(),
+        None => pivot_df
+            // No value columns provided, use remaining columns
+            .get_column_names()
+            .into_iter()
+            .map(|s| s.to_string())
```

Comment:
If you need a `Vec<String>`, I think you can use `get_column_names_owned` instead.

Reply:
Hi @stinodego, is there a more concise way than what I have here?

```rust
let all_columns = PlHashSet::<String>::from_iter(
    pivot_df
    .get_column_names_owned()
    .into_iter()
    .map(|s| s.to_string()));
```

`get_column_names_owned` returns a `Vec<SmartString>`, not sure if this involves any copying.

Reply:
Ok I think I found a better way, incoming soon.